src/components/Wizard/CreateVmWizard/StorageTab.js

```diff
@@ -0,0 +1,404 @@
+          isBootable: false,
+          editable: true,
+          edit: true, // trigger immediate edit
+          renderConfig,
+          ...values
+        }
+      ]
+    }));
+  };
+
+  createDisk = () => this.create(0);
+
+  attachStorage = () => this.create(1, { attachStorage: {} });
+
+  render() {
+    const columns = [
+      {
+        header: {
+          label: 'Disk Name',
+          props: {
+            style: {
+              width: '50%'
+            }
+          }
+        },
+        property: 'name',
+        hasAddendum: true,
+        renderConfigs: [
+          {
+            id: 'name-edit',
+            type: 'text'
+          },
+          {
+            id: 'name-attach-edit',
+            type: 'dropdown',
+            choices: this.props.storages.map(getName),
+            initialValue: '--- Select Storage ---'
+          }
+        ]
+      },
+      {
+        header: {
+          label: 'Size (GB)',
+          props: {
+            style: {
+              width: '23%'
+            }
+          }
+        },
+        property: 'size',
+        renderConfigs: [
+          {
+            id: 'size-edit',
+            type: 'positive-number'
+          }
+        ]
+      },
+      {
+        header: {
+          label: 'Storage Class',
+          props: {
+            style: {
+              width: '23%'
+            }
+          }
+        },
+        property: 'storageClass',
+
+        renderConfigs: [
+          {
+            id: 'storage-edit',
+            type: 'dropdown',
+            choices: this.props.storageClasses.map(getName),
+            initialValue: '--- Select ---'
+          }
+        ]
+      },
+      {
+        header: {
+          props: {
+            style: {
+              width: '4%'
+            }
+          }
+        },
+
+        renderConfigs: Array(3).fill({
+          id: 'actions',
+          type: ACTIONS_TYPE,
+          actions: [
+            {
+              actionType: DELETE_ACTION,
+              text: 'Remove Disk'
+            }
+          ],
+          visibleOnEdit: false
+        })
+      }
+    ];
+
+    // TODO: add create disk button once the create blank disk feature is implemented in kubevirt
+    /*
+     * <Button className="create-disk" onClick={this.createDisk} disabled={this.state.editing} id="create-disk-btn">
+     *   Create Disk
+     * </Button>
+    */
+    return (
+      <React.Fragment>
+        <ButtonGroup className="pull-right actions">
+          <Button onClick={this.attachStorage} disabled={this.state.editing} id="attach-storage-btn">
+            Attach Storage
+          </Button>
+        </ButtonGroup>
+        <EditableDraggableTable columns={columns} rows={this.state.rows} onChange={this.onChange} />
+      </React.Fragment>
+    );
+  }
+}
+
+StorageTab.defaultProps = {
+  initialDisks: []
+};
+
+StorageTab.propTypes = {
+  storageClasses: PropTypes.array.isRequired,
+  storages: PropTypes.array.isRequired,
```

Comment:
can you rename it to pvcs/volumeclaims/persistentvolumeclaims so its clearer what it actually represents ? Applyes also for other places.

Reply:
fixed